App/components/display-grain-state.jsx

```diff
@@ -0,0 +1,34 @@
+const React = require('react')
+
+module.exports = class DisplayGrainState extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    componentDidMount() {
+        var ace = window.ace;
+        ace.config.set('basePath', 'https://pagecdn.io/lib/ace/1.4.14/');
```

Comment:
ACE seems like quite a heavy dependency for providing markup to some JSON. Maybe we could just use a `<pre />` or a component like https://www.npmjs.com/package/react-json-pretty ?

Reply:
Changed to use <pre />